notebooks/advanced_RAG.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>Can we use the chat template feature? <a href="https://huggingface.co/docs/transformers/v4.37.0/en/chat_templating#templates-for-chat-models" target="_blank">https://huggingface.co/docs/transformers/v4.37.0/en/chat_templating#templates-for-chat-models</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Done!